TechnologyAssessmentApp/TechnologyAssessmentApp/Controllers/MasterTableViewController.swift

```diff
@@ -26,8 +26,11 @@ class MasterTableViewController: UITableViewController {
     //
     var isDetailViewController: Bool = true
```

Comment:
Yes, actually by default on iphone - splitViewController opens MasterViewController on first run.
To avoid that this bool is created. 
To pass in delegate method of SplitViewController and to control appearance of masterviewController